static/app/utils/array/lastOfArray.tsx

```diff
@@ -0,0 +1,5 @@
+export function lastOfArray<T extends Array<unknown> | ReadonlyArray<unknown>>(
+  t: T
+): T[number] {
+  return t[t.length - 1];
```

Comment:
this could probably be `t.at(-1)` and not even be a helper function, just do that inline. In both cases, the return value is correct with the current code, or with that change.

The benefit if any is that this is something people can grep for and import, instead of knowing about `.at()` ... or instead of typing the variable name twice.